tutorials/synthetic/synthetic.py

```diff
@@ -47,14 +47,29 @@
 
 #%%
 # Compute the true (empirical) LF accuracies
+# Shifting {1,2} -> {-1,1} for convenience here
+# TODO: Replace this with a utility function
 Ld = np.array(L.todense())
-Y = np.array([d.y for d in data])
-accs = Ld.T @ Y / n
-accs
+Ld[Ld == 2] = -1
+Y = np.array([1 if d.y == 1 else -1 for d in data])
+accs_emp = 0.5 * Ld.T @ Y / n + 0.5
 
 #%%
 # Run the LabelModel to estimate the LF accuracies
-# TODO
+label_model = LabelModel()
+label_model.train_model(L)
+accs_est = label_model.get_accuracies()
+print(f"Avg. LF accuracy estimation error: {np.mean(np.abs(accs_emp - accs_est))}")
+
+#%%
+# Get probabilistic training labels
+Y_prob = label_model.predict_proba(L)
+
+# Get the hard-thresholded predictions and compute the accuracy
```

Comment:
These utility function (prob_to_pred() and pred_to_prob()) are in snorkel/mtl/utils.py at the moment, and can be moved up a level, since they're not mtl-specific.

Reply:
Will add utils in

Reply:
Replaced all the generic steps with utils.

Side note: I almost started cleaning up and merging all the util files... but left that for another PR.